hotel_reservation/views/hotel_reservation.xml

```diff
@@ -60,6 +60,10 @@
                             <field name="checkout"/>
                             <field name="adults" required="1"/>
                             <field name="children"/>
+                            <field name="open"/>
+                        </group>
+                        <group>
+                            <field name="note"/>
```

Comment:
I'd put this note in the "note group" and add a name attribute to that group.

I'll put my housekeeping note in that group as well.

Reply:
Done! Your housekeeping note is welcome to join!